tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -1,4 +1,7 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿#if Net70App
+#define NET70
```

Comment:
You don't need to define your own TFM flags; [.NET already includes them](https://github.com/dotnet/designs/blob/main/accepted/2020/net5/net5.md#preprocessor-symbols).